app/src/main/res/values/strings.xml

```diff
@@ -60,4 +60,6 @@
     <string name="date_caption_for_locked_achievements">Last updated</string>
     <string name="date_caption_for_unlocked_achievements">Earned</string>
     <string name="steps_completed">Steps completed</string>
+    <string name="loading_achievements_exception">Failed to load achievements from client: </string>
```

Comment:
can we move this string back where it was?

Usually, we would have strings which appear in the components on the UI like "steps completed".
However, this string is used only for debugging purposes so we can can it in the java file